src/domain/relay/limit-addresses.mapper.ts

```diff
@@ -228,28 +228,14 @@ export class LimitAddressesMapper {
     chainId: string;
     address: `0x${string}`;
   }): boolean {
-    const multiSendCallOnlyDeployment = getMultiSendCallOnlyDeployment({
-      version: args.version,
-      network: args.chainId,
-    });
-
-    const isCallOnly =
-      multiSendCallOnlyDeployment?.networkAddresses[args.chainId] ===
-        args.address ||
-      multiSendCallOnlyDeployment?.defaultAddress === args.address;
+    const multiSendCallOnlyDeployments = getMultiSendCallOnlyDeployments(args);
+    const isCallOnly = multiSendCallOnlyDeployments.includes(args.address);
 
     if (isCallOnly) {
       return true;
     }
 
-    const multiSendCallDeployment = getMultiSendDeployment({
-      version: args.version,
-      network: args.chainId,
-    });
-    return (
-      multiSendCallDeployment?.networkAddresses[args.chainId] ===
-        args.address || multiSendCallDeployment?.defaultAddress === args.address
-    );
+    return getMultiSendDeployments(args).includes(args.address);
```

Comment:
Nit: we could just

```typescript
return getMultiSendCallOnlyDeployments(args).includes((args.address) || getMultiSendDeployments(args).includes(args.address);
```

And therefore get rid of all the variables and the `if` statement, right? Do you think it would be less readable?